osquery/core/conversions.cpp

```diff
@@ -385,10 +385,11 @@ std::vector<std::string> split(const std::string& s, const std::string& delim) {
 }
 
 std::vector<std::string> split(const std::string& s,
-                               const std::string& delim,
+                               char delim,
```

Comment:
The objective is that when delim.size() > 1 this code has bug. I can fix this, but as there was no use of this function with more than one delimiter, just corrected signature, so noone will have bug because of that.

https://github.com/guliashvili/osquery/blob/c9ecc939cd2e194823a28a1e17f8619736b566a6/osquery/core/conversions.cpp#L405 The bug is here(joining on all the delimiters)

Reply:
@muffins 
Does this answer your question?